create-ava-rule.js

```diff
@@ -19,9 +19,66 @@ const avaImportDeclarationAst = {
 		type: 'Literal',
 		value: 'ava'
 	}
-};
+}, {
+	type: 'ImportDeclaration',
```

Comment:
Not happy with the wall of ASTs... any way to make this easier to comprehend?

Reply:
You might have been able to reduce the amount of AST code if you did the checks programmatically instead of comparing the AST tree, but that will also be more fragile and IMHO less readable, so I think this is fine.

Reply:
Some comments here and there to define what each AST match against could help.